.github/command-docs.md

```diff
@@ -100,6 +100,7 @@
 | Command           | Description                                                                                                                                                                                | Alias                                                       |
 |:------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------|
 | .botissue         | Provides details on where to create a FBW Discord Bot Issue or Feature Request                                                                                                             | .bot-issue <br> .botfeature <br> .bot-feature <br> .botfeat |
+| .devlanguages     | Provides a list of development languages used in the A32NX                                                                                                                                 | .devlanguage <br> .devlang <br> .dl                         |
```

Comment:
Should be before .botissue to be in line with the alphabetical order used in the command-docs.md

Reply:
D comes after B?

Reply:
Yeah no brain stopped working